Modules/IO/ImageBase/include/itkImageFileWriter.hxx

```diff
@@ -209,8 +209,7 @@ ImageFileWriter<TInputImage>::Write()
   // confiugure pixel type
   if (strcmp(input->GetNameOfClass(), "VectorImage") == 0)
   {
-    using VectorImageScalarType = typename InputImageType::InternalPixelType;
-    m_ImageIO->SetPixelTypeInfo(static_cast<const VectorImageScalarType *>(nullptr));
+    m_ImageIO->SetPixelTypeInfo(static_cast<const InputImagePixelType *>(nullptr));
```

Comment:
I'd say that the `else` block is no longer necessary, since the above statement does also appear in [here](https://github.com/InsightSoftwareConsortium/ITK/pull/2335/files#diff-5c9466ac18688fab7c93762bd2a2542f7cb9ca089f106465d17534b6ecb7241bL220).

Reply:
:+1: good call

Reply:
Just to avoid any misunderstanding, and since I realize my comment may lead to confusion: I meant that since the statement can be taken out of the `if/else` block, the `else` block would no longer be necessary if that were done. Sorry. I think people understood :+1:, but just to be clear.